### ISSUE TITLE
Missing Feature

### ISSUE THREAD

**Issue by DrJaJa** [MrPig91/PSChiaPlotter#52]
Could you add one of Swar's Plotter Features like "max_concurrent_with_start_early" and "concurrency_start_early_phase"?

I've found out that I can run 2 K32 plots in parallel in a 476GB NVMe Drive if the second one starts just when the first one enters the 3rd phase.  Same with the 1TB drive (4 instead of 3 with the same parameter).

**Comment by MrPig91** [comment]
I will look into this, I think this will be the first big update to the plotting manager. Thank you for the feedback!

**Comment by DrJaJa** [comment]
Also, adding the option to stagger the plots by percentage value will allow us to fine-tunning even more the jobs because, in this application, percentage is a consistent marker, time is not.

**Comment by MrPig91** [comment]
Phase 1 limitor has now been added and is in version 1.0.32. For sake simplicity and less things breaker I will not be adding stagger by %. I added a bunch of features to this update so if you come across any issues please create a new issue.
